runtime/doc/lsp.txt

```diff
@@ -137,7 +137,18 @@ FAQ                                                     *lsp-faq*
      " Auto-format *.rs (rust) files prior to saving them
      " (async = false is the default for format)
      autocmd BufWritePre *.rs lua vim.lsp.buf.format({ async = false })
+<
+- Q: How can I disable LSP formatting when using the |gq| command?
+  A: First you could try using |gw| as an alternative to |gq| .
+  If that is not good enough for you, then you can use the following 
+  lua snippet to completely disable LSP formatting using |gq| :
```

Comment:
no need to link |gq| every time it is mentioned, use "gq" after the first time. Else it's visually noisy.

Reply:
Reduced noise.